### DISCUSSION TITLE
Game gear dumps

### DISCUSSION THREAD

**Discussion by mew4900** [sanni/cartreader#349] [Need help]
Is it possible to dump gear games using the sega master system settings and a adapter like the 3-in-1 hyperkin adapter or the retrofreak adapter that fits into the genesis/mega drive slot?  I have a retron 5 and I can play game gear games using it but didn’t know about dumping them. Thanks for any help.

**Comment by sanni** [comment]
I'm not sure if anyone has tried this yet. Most likely the Arduino code would need to be changed a little so the Cart Reader knows how the adapter is wired. Do you have a picture of the PCB inside of those adapters(front/back)?

**Reply by mew4900** [reply]
I’m sorry for the late reply. I took the screws off mine and was able to get some pictures. I only got one side for fear if I took it apart more I may break it (I’m not ver knowledgable on this kind of thing). Here’s what I got:
![A35475E3-1F8A-499C-920E-F18640E4BF39](https://user-images.githubusercontent.com/101483250/159147671-8f6ac92c-d4e0-4515-944d-e87184f87fe7.jpeg)
![F9C90350-1B08-4905-BD5D-F7F649444CA2](https://user-images.githubusercontent.com/101483250/159147674-e063e131-dc2b-4283-adb9-d963fbab7551.jpeg)
![7F568FAC-1FAB-44BF-ADA6-6A916B99079C](https://user-images.githubusercontent.com/101483250/159147675-3b08346a-7c1e-46fe-91ac-1325c512a375.jpeg)
![86E38DF9-6046-4EA8-BA56-8D817969EDC2](https://user-images.githubusercontent.com/101483250/159147676-965e59c5-9d7c-4b8a-97c9-cd3d2feb5223.jpeg)

Does this help at all?

**Reply by mew4900** [reply]
Would there be any harm to the reader, adapter, or cartridge in trying to read a game gear game with this type of adapter?

**Reply by sanni** [reply]
Sadly just one side of the PCB doesn't tell me anything about the pinout of the adapter. 
Also the STM chip makes it more complicated since there will be code inside it doing some sort of mapping.

If I were you I would just wait until someone else tries it first, especially considering the current going rate of that adapter.

**Reply by mew4900** [reply]
Thank you so much. It means a lot to me that you would help a newbie like me. I am definitely taking your advice.
